js/adapt-assessmentArticleModel.js

```diff
@@ -728,6 +731,7 @@ const AssessmentModel = {
     blocks.forEach((block, blockIndex) => {
       const blockQuestions = block.findDescendantModels('question');
       blockQuestions.forEach((question, questionIndex) => {
+        if (_questions.some(_question => _question._id === question.get('_id'))) return;
```

Comment:
Again I don't think a question uniqueness check is require here as the block will be unique at his point.